pkgs/by-name/ta/tabby-terminal/fix-deps-for-new-electron.patch

```diff
@@ -0,0 +1,434 @@
+diff --git a/app/package.json b/app/package.json
```

Comment:
Hmm. Patching a lot of lockfiles in a very active project feels perilous. Maybe we should just wait for upstream to update?